pegasus/helpers/section_api_helpers.rb

```diff
@@ -216,8 +216,7 @@ def self.create(params)
     created_at = DateTime.now
 
     row = nil
-    tries = 0
-    begin
+    retryable(tries: 2, sleep: 0, on: Sequel::UniqueConstraintViolation) do
```

Comment:
to test, go to teacher dashboard at http://code.org/teacher-dashboard and create a section